src/open_inwoner/pdc/tests/test_data_migrations.py

```diff
@@ -0,0 +1,47 @@
+from open_inwoner.utils.tests.test_migrations import TestMigrations
+
+
+class TruncateProductSummaryMigrationTests(TestMigrations):
+    app = "pdc"
+    migrate_from = "0044_question_category_or_product_null"
+    migrate_to = "0045_auto_20230220_1628"
```

Comment:
Shouldn't this test to 0046_ ?

Reply:
I wanted to test that the summary is truncated, that's why I tested the initial (0044_) compared to the data migration (0045_). If I am wrong let me know please.